### PR TITLE
Remove OF from build, comments, package.json, etc

### PR DESCRIPTION
**Pull request by carterbs**
[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/23020/details)

* See card for details

### CODE REVIEW

File: configs/application/config.json [hunk 1]
```diff
@@ -58,7 +61,7 @@
 			"unknownComponent": "nonConfiguredComponent"
 		},
 		"assimilation": {
-			"useOpenFinSpawn": false,
+			"useFeaSpawn": false,
```
Comment (zona-ciq) [inline configs/application/config.json]: In assimilation window manager we're still using useOpenFinSpawn. This will cause that code to break

Comment (carterbs) [reply configs/application/config.json]: Not in 5.0.0

Comment (carterbs) [reply configs/application/config.json]: https://github.com/ChartIQ/finsemble/commit/c1f36d12d96aeaac29b9854d9c0ef04bbfdb44df

Comment (zona-ciq) [reply configs/application/config.json]: Cool!

**Pull request merged**